opencti-platform/opencti-graphql/src/manager/notificationManager.ts

```diff
@@ -151,7 +151,8 @@ const generateAssigneeTrigger = (user: AuthUser) => {
     filterGroups: []
   };
   return {
-    name: 'Default Trigger for Assignee/Participant',
+    internal_id: `default-trigger-${user.id}`,
```

Comment:
Ok for that fix that was needed by I think this is not really aligned with the initial issue